src/pages/ListPage/ListPage.tsx

```diff
@@ -55,12 +57,28 @@ const getPoliciesFromPayload = (payload: ReturnType<typeof useGetListPagePolicie
     return undefined;
 };
 
+const INVENTORY_TOTAL_FETCH_URL = '/api/inventory/v1/hosts';
+
 const ListPage: React.FunctionComponent<unknown> = () => {
 
     const [ policyWizardState, setPolicyWizardState ] = React.useState<PolicyWizardState>({
         isOpen: false
     });
 
+    const [ hasSystems, setHasSystems ] = React.useState(true);
+
+    React.useEffect(() => {
+        try {
+            axios
+            .get(`${INVENTORY_TOTAL_FETCH_URL}?page=1&per_page=1`)
+            .then(({ data }) => {
+                setHasSystems(data.total > 0);
+            });
+        } catch (e) {
+            console.log(e);
+        }
+    }, [ hasSystems ]);
```

Comment:
You don't need this dependency.

```suggestion
    }, [ setHasSystems ]);
```